[CS2113T-T09-2] MoneyGoWhere

nigelhao opened the pull request:
MoneyGoWhere is a point-of-sale, desktop app for tracking sales and inventory, optimised for use via a Command line Interface (CLI). With the ability to type fast and with familiarity of the system, you will be able to quickly track sales, check inventory and calculate change.

varnika1402 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,506 @@
+* `/additem -n <item_name> -p <price>`
+
+This sequence diagram models the interaction between various components in MoneyGoWhere when the user inputs the 
+command `/additem`.
```
Can try including at least one sequence or class diagram?

varnika1402 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,506 @@
+## Requirements
 
-## Non-Functional Requirements
+### Project Scope
```
Target user and value proposition could be included to understand the application idea better.

varnika1402 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,506 @@
+   the list of `Transactions`.
+
+<hr>
+
```
I like how the workflow of each command is well-elaborated.

varnika1402 reviewed (commented):
Overall good explanation of commands but can include other important details like target user profile and sequence diagrams etc. in the DG too.

RuiShengGit commented on docs/images/developersGuide/addItem.png:
There seems to be a repeat of calling the method mapArgumentAlias from Command, maybe just have one arrow for it

RuiShengGit commented on docs/images/developersGuide/addItem.png:
I think when the class self invokes a method, the arrow should be pointing to the top of the activation bar and not to the middle like for checkFlags().

RuiShengGit commented on docs/images/developersGuide/payment.png:
For the PaymentValidation class, for methods that are self-invoked, the arrow should be from the larger activation bar to the start of the smaller activation bar

RuiShengGit commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
 # Developer Guide
 
+## Table of Contents
+
```
Can consider standardizing the diagrams format as there are variations to the diagrams and can make the diagrams difficult to read.

RuiShengGit commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+  * [Basic Mode](#basic-mode-find-an-item)
+
+<hr>
+
```
Consider adding object diagrams for each steps of the feature to give users a better idea on how the feature works.

MuxPotato commented on docs/images/developersGuide/addItem.png:
For the validateFlags() function, the opt block [invalid flags] can include an return arrow from :AdditemValidation back to :Menu before the menu returns control to user

MuxPotato replied:
The same can be applied to validateCommand()

MuxPotato commented on docs/images/developersGuide/addOrder.png:
For the :AddOrderValidation and :AddMultipleOrderValidation, is the lifeline supposed to end at the end of this function?
I am unsure of this because, this those two objects are to be referenced everytime "addOrder" is called, but it is not suggested by the sequence diagram that they a new object is initialized everytime "addOrder" is called. This would potentially cause a bug in the code as it tries to reference an object that is not there.

MuxPotato commented on docs/images/developersGuide/addOrderValidation.png:
is findItemIndex supposed to print out an error message to the user? 
If it is, I think it can be named to something like "printErrorItemIndex"

MuxPotato replied:
The lifeline of :AddOrderValidation and :Menu ending is potentially wrong as stated earlier

MuxPotato commented on docs/images/developersGuide/itemPackage.png:
Very nice class-level diagram

sansders committed:
Merge pull request #216 from sansders/master

Edit typo in DG

MuxPotato commented on docs/images/developersGuide/payment.png:
For the activation bar for setPaymentType and setStatus, a return arrow can be used going from :Order to :Payment, for better clarification

MuxPotato reviewed (commented):
Overall, good sequence diagrams. 
The use of ref block to improve readability while keeping attention to detail was excellent.

natashatanyt committed:
Update minor typos and styling issues

natashatanyt committed:
Merge pull request #217 from natashatanyt/master

Update minor typos and styling issues

ZIZI-czh commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+calls `Menu#appendItem` on the new `Item` object to add it to `ArrayList<Item> items` in `Menu`. 
+Then, `Menu#save` is invoked to save the changes to the local storage file.
+1. `Router` object then calls `MenuUi#printCommandSuccess` to print a message indicating that the item has been 
+successfully added to the menu.
```
Maybe can add in extra feature to allow the user to go back to the table content?

ZIZI-czh commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+
+<hr>
+
+#### List all Items
```
May be you want to include a sequence to explain this features because it is hard to understand?

ZIZI-czh commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+* `additem`
+* `1`
+
+This sequence diagram models the interaction between various components in MoneyGoWhere when the user inputs the 
```
Maybe you want to include another sequence diagram for this feature to differentiate from the advanced model?

ZIZI-czh commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+### Report Commands
 
-### Value proposition
+#### Overview
```
You may want to the value proposition and target users for the product

sansders committed:
Update sequence diagrams in DG

sansders committed:
Merge pull request #218 from sansders/master

Update sequence diagrams in DG

JuneNita commented on docs/images/developersGuide/addMultipleAddOrders.png:
Nice and clean UML but I think the activation bar is missing.

JuneNita commented on docs/images/developersGuide/payment.png:
The PaymentValidation's 2nd order activation bar might be needed to shift to the left a bit to make it at the middle.

JuneNita commented on docs/images/developersGuide/payment.png:
Activation bar is too long. Can reduce the activation bar length or shift the arrow.

JuneNita commented on docs/images/developersGuide/payment.png:
The X representing the deletion of an object should be at the end of lifeline.

JuneNita commented on docs/images/developersGuide/AssistedAddOrder.png:
I think that the activation bar is missing.

JuneNita reviewed (commented):
Generally, the UML can be improved.
Many activation bars are missing.

liuziyang020319 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+successfully added to the menu.
+
+
+##### Basic Mode [Delete an Item]()
```
There is a strange hyperlink in the website. I think it is maybe a mistake in typing.

liuziyang020319 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
 # Developer Guide
```
Overall, I like the structure and the lay out of the DG. However, I think it could be better to standardizing the diagrams.

liuziyang020319 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+
+#### Add an Item
+
+##### Advanced Mode Add an Item
```
It could be better if you add some sequence diagram to illustrate!

liuziyang020319 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+This sequence diagram models the interaction between various components in MoneyGoWhere when the user inputs the 
+command `/additem`.
+
+![](./images/developersGuide/addItem.png)
```
ui.promptUserInput() recives the input of the User. Maybe the "additem order" sent by User should be sent to the UI？

liuziyang020319 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,527 @@
+   Point-of-Sale (POS) system.
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+### Glossary
```
I find that there are many things that have not finished yet. Hopefully you won't forget this part! Overall, that's a good job!👍

sansders committed:
Add logo

sansders committed:
Add logo

nigelhao committed:
Update code quality and unit tests

nigelhao committed:
Update javadocs quality

nigelhao committed:
Update code quality

nigelhao committed:
Update code quality again

nigelhao committed:
Update menu and transaction constructors

nigelhao committed:
Merge pull request #222 from nigelhao/master

Update code quality

nigelhao committed:
Update file storage exception handling

nigelhao committed:
Update code quality

nigelhao committed:
Remove unused imports from store

nigelhao committed:
Update StoreUi

nigelhao committed:
Update StoreUi again

nigelhao committed:
Update code quality

nigelhao committed:
Merge pull request #257 from nigelhao/master

Update file storage exception handling

nigelhao committed:
Remove invalid command exception message.

nigelhao committed:
Merge pull request #258 from nigelhao/master

Remove invalid command exception message.

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp: (22 commits)
  Remove invalid command exception message.
  Update code quality
  Update StoreUi again
  Update StoreUi
  Remove unused imports from store
  Update code quality
  Update file storage exception handling
  Update menu and transaction constructors
  Update code quality again
  Update code quality
  Update javadocs quality
  Update code quality and unit tests
  Update sequence diagrams in DG
  Update minor typos and styling issues
  Edit typo in DG
  Add more partitions to UG
  Create _config.yml
  Update backslash to frontslash in image paths of DG
  Add images
  Add sequence diagrams to order's portion of the DG
  ...

yuanners99 committed:
Fix payment bug

yuanners99 committed:
Remove duplicate successful add order statement

chungnicholas committed:
Merge pull request #259 from yuanners/yuan

Fix payment bugs

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:

yuanners99 committed:
Fix refund order bug

chungnicholas committed:
Merge pull request #260 from yuanners/yuan

Fix refund order bugs

chungnicholas committed:
Refactor `MenuAssistant.java` to show user how to abort operation

chungnicholas committed:
Refactor formatting

chungnicholas committed:
Merge pull request #34 from NicholasChungJunJie/NicholasChungJunJie

Refactor `MenuAssistant.java` to show user how to abort operation

chungnicholas committed:
Remove whitespace around name and refactor output for better readability

yuanners99 committed:
Merge pull request #261 from NicholasChungJunJie/master

Refactor `MenuAssistant.java` to show user how to abort operation

chungnicholas committed:
Merge pull request #35 from NicholasChungJunJie/NicholasChungJunJie

Remove whitespace around name and refactor output for better readability

chungnicholas committed:
Refactor name validation to reject pure numbers

yuanners99 committed:
Merge pull request #262 from NicholasChungJunJie/master

Remove whitespace around name and refactor output for better readability

chungnicholas committed:
Merge pull request #36 from NicholasChungJunJie/NicholasChungJunJie

Refactor name validation to reject pure numbers

yuanners99 committed:
Merge pull request #263 from NicholasChungJunJie/master

Refactor name validation to reject pure numbers

nigelhao committed:
Update all sc.nextline() to ui.inputHandler()

nigelhao committed:
Update code quality

nigelhao committed:
Merge commit 'e6a9cec49207c4cc3fd8eb9953b276a614e710cc'

* commit 'e6a9cec49207c4cc3fd8eb9953b276a614e710cc':
  Refactor name validation to reject pure numbers
  Remove whitespace around name and refactor output for better readability
  Refactor formatting
  Refactor `MenuAssistant.java` to show user how to abort operation
  Fix refund order bug
  Remove duplicate successful add order statement
  Fix payment bug

nigelhao committed:
Update variable visibility

nigelhao committed:
Remove unused variables and methods

nigelhao committed:
Merge pull request #264 from nigelhao/master

Update code quality

natashatanyt committed:
Fix double printing finditem success

natashatanyt committed:
Trim user input to prevent unwanted errors

natashatanyt committed:
Remove unused imports

natashatanyt committed:
Remove unused imports

natashatanyt committed:
Merge branch 'master' of https://github.com/natashatanyt/tp

natashatanyt committed:
Merge pull request #265 from natashatanyt/master

Bug Fix

chungnicholas committed:
Fix duplicate success message in `finditem`

chungnicholas committed:
Merge pull request #37 from NicholasChungJunJie/NicholasChungJunJie

Fix duplicate success message in `finditem`

yuanners99 committed:
Merge pull request #266 from NicholasChungJunJie/master

Fix duplicate success message in `finditem`

sansders committed:
Merge remote-tracking branch 'origin/master'

sansders committed:
Update noob order validation

sansders committed:
Fix checkstyle

sansders committed:
Merge pull request #267 from sansders/master

Update noob order validation

nigelhao committed:
Fix duplicate options in command issue

nigelhao committed:
Merge commit 'dee1cbac41e1f080146e28a7f4170a99c4c25cc6'

* commit 'dee1cbac41e1f080146e28a7f4170a99c4c25cc6':
  Fix checkstyle
  Update noob order validation
  Fix duplicate success message in `finditem`
  Add logo
  Remove unused imports
  Remove unused imports
  Trim user input to prevent unwanted errors
  Fix double printing finditem success
  Add logo

nigelhao committed:
Fix error in command class

nigelhao committed:
Merge pull request #268 from nigelhao/master

Fix duplicate options in command issue

natashatanyt committed:
Increase padding for printing price when using `listitem`

natashatanyt committed:
Reformat `help`, add `/help`

natashatanyt committed:
Change minor formatting issues

natashatanyt committed:
Merge pull request #269 from natashatanyt/master

Add new `help` command, Change `listitem` formatting

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp: (22 commits)
  Change minor formatting issues
  Reformat `help`, add `/help`
  Increase padding for printing price when using `listitem`
  Fix error in command class
  Fix duplicate options in command issue
  Fix checkstyle
  Update noob order validation
  Fix duplicate success message in `finditem`
  Add logo
  Remove unused imports
  Remove unused imports
  Trim user input to prevent unwanted errors
  Fix double printing finditem success
  Remove unused variables and methods
  Update variable visibility
  Update code quality
  Update all sc.nextline() to ui.inputHandler()
  Refactor name validation to reject pure numbers
  Remove whitespace around name and refactor output for better readability
  Refactor formatting
  ...

yuanners99 committed:
Fix refundorder pro mode bug

yuanners99 committed:
Fix checkstyle

yuanners99 committed:
Update PPP

yuanners99 committed:
Fix checkstyle

yuanners99 committed:
Add exceptions to method signature

chungnicholas committed:
Merge pull request #272 from yuanners/yuan

Fix refundorder pro mode bug

yuanners99 committed:
Update DG on payment command

chungnicholas committed:
Merge pull request #273 from yuanners/yuan

Update DG on payment command

sansders committed:
Remove unused sequence diagrams

sansders committed:
Add refundOrder sequence diagram to DG image folder

sansders committed:
Fix checkstyle

How'd that get in there?

chungnicholas committed:
Invalidate extra arguments

chungnicholas committed:
Invalidate extra arguments

chungnicholas committed:
Merge pull request #38 from NicholasChungJunJie/NicholasChungJunJie

Invalidate extra arguments

yuanners99 committed:
Merge pull request #275 from NicholasChungJunJie/master

Invalidate extra arguments

nigelhao committed:
Update development guide for the design section

nigelhao committed:
Merge pull request #276 from nigelhao/master

Update development guide for the design section

sansders committed:
Remove .puml files

What are they good for?

sansders committed:
Move files into folders, add seq. diagrams for statistic

sansders committed:
Add sequence diagrams for deleteItem, findItem, listItem, and updateItem

sansders committed:
Merge pull request #274 from sansders/master

Add more sequence diagrams to the DG

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Add sequence diagrams for deleteItem, findItem, listItem, and updateItem
  Move files into folders, add seq. diagrams for statistic
  Remove .puml files
  Update development guide for the design section
  Invalidate extra arguments
  Invalidate extra arguments
  Fix checkstyle
  Add refundOrder sequence diagram to DG image folder
  Remove unused sequence diagrams

chungnicholas committed:
Add info

chungnicholas committed:
Merge pull request #39 from NicholasChungJunJie/NicholasChungJunJie

Add info

chungnicholas committed:
Refactor EXPECTED.txt

chungnicholas committed:
Merge pull request #40 from NicholasChungJunJie/NicholasChungJunJie

Refactor EXPECTED.txt

yuanners99 committed:
Merge pull request #277 from NicholasChungJunJie/master

Add info to profile page

yuanners99 committed:
Add sequence diagrams to DG

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Refactor EXPECTED.txt
  Add info

chungnicholas committed:
Merge pull request #278 from yuanners/yuan

Add sequence diagrams to DG

sansders committed:
Add PPP

sansders committed:
Merge pull request #279 from sansders/master

Add PPP for sansders

sansders committed:
Add PPP

sansders committed:
Merge pull request #280 from sansders/master

Add PPP

sansders committed:
Update aboutUs.md

sansders committed:
Merge pull request #281 from sansders/master

Update aboutUs.md

sansders committed:
Update PPP

sansders committed:
Merge pull request #282 from sansders/master

Update PPP

natashatanyt committed:
Add PPP

natashatanyt committed:
Merge pull request #283 from natashatanyt/master

Add PPP

sansders committed:
Add additional tests

sansders committed:
Remove unused import

sansders committed:
Merge pull request #284 from sansders/master

Add additional tests

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Remove unused import
  Add additional tests
  Add PPP
  Update PPP
  Update aboutUs.md
  Add PPP
  Add PPP

yuanners99 committed:
Add junit test cases for refund order and payment commands

yuanners99 committed:
Update DG on the make payment basic mode

yuanners99 committed:
Remove unused imports

chungnicholas committed:
Merge pull request #285 from yuanners/yuan

Update DG and Junit tests

chungnicholas committed:
Update PPP

chungnicholas committed:
Update PPP

chungnicholas committed:
Update PPP

chungnicholas committed:
Merge pull request #41 from NicholasChungJunJie/NicholasChungJunJie

Update PPP

yuanners99 committed:
Merge pull request #286 from NicholasChungJunJie/master

Update PPP

sansders committed:
Move test files into proper packages

sansders committed:
Remove unused imports

sansders committed:
Merge pull request #287 from sansders/master

Move test files into proper packages

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Remove unused imports
  Move test files into proper packages
  Update PPP
  Update PPP
  Update PPP

natashatanyt committed:
Update PPP

natashatanyt committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp

natashatanyt committed:
Merge pull request #288 from natashatanyt/master

Update PPP

yuanners99 committed:
Fix empty order list bug

yuanners99 committed:
Add glossary, missing content portions and instructions for manual testing

yuanners99 committed:
Add hyperlinks to glossary

yuanners99 committed:
Update PPP

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Update PPP

yuanners99 committed:
Update PPP

chungnicholas committed:
Merge pull request #289 from yuanners/yuan

Update DG and PPP

chungnicholas committed:
Fix formatting bugs

chungnicholas committed:
Add info to PPP

chungnicholas committed:
Merge pull request #42 from NicholasChungJunJie/NicholasChungJunJie

Fix formatting bugs in DG

yuanners99 committed:
Merge pull request #290 from NicholasChungJunJie/master

Fix formatting bugs in DG

natashatanyt committed:
add junit test for items

natashatanyt committed:
Merge pull request #291 from natashatanyt/master

Add junit test for items

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  add junit test for items
  Add info to PPP
  Fix formatting bugs

yuanners99 committed:
Update acknowledgement section

yuanners99 committed:
Add new Junit test cases for payment and refund

chungnicholas committed:
Merge pull request #292 from yuanners/yuan

Add more Junit test cases

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:

yuanners99 committed:
Add Junit test for refund order

chungnicholas committed:
Merge pull request #293 from yuanners/yuan

Add Junit test case for refund order

chungnicholas committed:
Fix bugs in `ItemTest`

chungnicholas committed:
Merge pull request #43 from NicholasChungJunJie/NicholasChungJunJie

Fix bugs in `ItemTest`

yuanners99 committed:
Merge pull request #294 from NicholasChungJunJie/master

Fix bugs in `ItemTest`

natashatanyt committed:
Change headers to follow UG naming scheme

natashatanyt committed:
Merge pull request #295 from natashatanyt/master

Change headers to follow UG naming scheme

sansders committed:
Fix pro add single order bugs

sansders committed:
Fix bug in assisted addorder

Fixed a application crash bug when an empty input is provided in assisted addorder prompts

sansders committed:
Merge pull request #296 from sansders/master

Fix addorder bugs

natashatanyt committed:
fix finditem bug

natashatanyt committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp

natashatanyt committed:
Merge pull request #297 from natashatanyt/master

Fix finditem bug

chungnicholas committed:
Fix failing CI

chungnicholas committed:
Merge pull request #44 from NicholasChungJunJie/NicholasChungJunJie

Fix failing CI

yuanners99 committed:
Merge pull request #298 from NicholasChungJunJie/master

Fix failing CI

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Fix failing CI
  fix finditem bug
  Fix bug in assisted addorder
  Fix pro add single order bugs
  Change headers to follow UG naming scheme
  Fix bugs in `ItemTest`

yuanners99 committed:
Update PPP

chungnicholas committed:
Merge pull request #299 from yuanners/yuan

Update PPP

chungnicholas committed:
Update PPP

chungnicholas committed:
Merge pull request #45 from NicholasChungJunJie/NicholasChungJunJie

Update PPP

sansders committed:
Update DG ToC

sansders committed:
Merge pull request #301 from sansders/master

Update DG ToC

yuanners99 committed:
Merge pull request #300 from NicholasChungJunJie/master

Update PPP

sansders committed:
Remove unused file

sansders committed:
Merge pull request #302 from sansders/master

Remove unused file

natashatanyt committed:
Add new UG

natashatanyt committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp

natashatanyt committed:
Merge pull request #303 from natashatanyt/master

Add new UG

chungnicholas committed:
Add `UpdateItem` to UG

chungnicholas committed:
Update PPP

chungnicholas committed:
Merge pull request #46 from NicholasChungJunJie/NicholasChungJunJie

Add `UpdateItem` section to UG

chungnicholas committed:
Merge pull request #304 from NicholasChungJunJie/master

Add `Update an Item` to UG

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Update PPP
  Add `UpdateItem` to UG
  Add new UG
  Remove unused file
  Update DG ToC
  Update PPP

nigelhao committed:
Update find an item user guide

nigelhao committed:
Merge pull request #305 from nigelhao/master

Update find an item user guide

yuanners99 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp into yuan

* 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp:
  Update find an item user guide

yuanners99 committed:
Update user guide on add order and payment section

yuanners99 committed:
Merge pull request #306 from yuanners/yuan

Update user guide on add order and payment sections

sansders committed:
Add order content into UG

sansders committed:
Merge pull request #307 from sansders/master

Add order content into UG

natashatanyt committed:
Fix bugs in DG

natashatanyt committed:
Merge pull request #308 from natashatanyt/master

Fix bugs in DG

natashatanyt committed:
Update UG

natashatanyt committed:
Merge pull request #309 from natashatanyt/master

Update UG

sansders committed:
Updated print statements to include newlines

sansders committed:
Fix logo

sansders committed:
Fix checkstyle

sansders committed:
Merge pull request #310 from sansders/master

Updated Orders' print statements to include a line break before/after every statement

sansders committed:
Fix broken icon

sansders committed:
Merge pull request #311 from sansders/master

Fix broken icon

sansders committed:
Fix broken icon

sansders committed:
Merge pull request #312 from sansders/master

Fix broken icon

sansders committed:
Fix broken icon

sansders committed:
Merge pull request #313 from sansders/master

Fix broken icon

sansders committed:
Fix broken icon

sansders committed:
Merge pull request #314 from sansders/master

Fix broken icon

sansders committed:
Fix broken icon

sansders committed:
Merge pull request #315 from sansders/master

Fix broken icon

sansders committed:
Update DG

sansders committed:
Merge pull request #316 from sansders/master

Update DG

sansders committed:
Update DG

Added forced page breaks after each section

sansders committed:
Merge pull request #317 from sansders/master

Update DG

sansders committed:
Improve DG formatting

sansders committed:
Merge pull request #318 from sansders/master

Improve DG formatting

natashatanyt committed:
Fix UG bugs

natashatanyt committed:
Merge pull request #319 from natashatanyt/master

Fix UG bugs

sansders committed:
Update DG

Removed broken <hr> tag in DG

sansders committed:
Merge pull request #320 from sansders/master

Update DG

natashatanyt committed:
Fix more UG bugs

natashatanyt committed:
Merge pull request #321 from natashatanyt/master

Fix more UG bugs

natashatanyt committed:
Fix more UG bugs

natashatanyt committed:
Merge pull request #322 from natashatanyt/master

Fix more UG bugs

natashatanyt committed:
Fix broken hyperlink

natashatanyt committed:
Merge pull request #323 from natashatanyt/master

Fix broken hyperlink

natashatanyt committed:
Change formatting

natashatanyt committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113T-T09-2/tp

natashatanyt committed:
Add page break

natashatanyt committed:
Merge pull request #324 from natashatanyt/master

Update formatting

natashatanyt committed:
Add another pagebreak

natashatanyt committed:
Merge pull request #325 from natashatanyt/master

Add another pagebreak

nigelhao committed:
Update for ppp draft

nigelhao committed:
Merge commit '3a1dec201d44423b5a0c2f11a87766c82ab92f21'

* commit '3a1dec201d44423b5a0c2f11a87766c82ab92f21': (23 commits)
  Add another pagebreak
  Add page break
  Change formatting
  Fix broken hyperlink
  Fix more UG bugs
  Fix more UG bugs
  Update DG
  Fix UG bugs
  Improve DG formatting
  Update DG
  Update DG
  Fix broken icon
  Fix broken icon
  Fix broken icon
  Fix broken icon
  Fix broken icon
  Fix checkstyle
  Fix logo
  Updated print statements to include newlines
  Update UG
  ...

nigelhao committed:
Merge pull request #326 from nigelhao/master

Update for PPP draft

sansders committed:
Add CLI tutorial to UG

sansders committed:
Update introduction

sansders committed:
Add more to the introduction in UG

sansders committed:
Merge pull request #327 from sansders/master

Update UG

natashatanyt committed:
Edit UG for 2101

natashatanyt committed:
Add hr

natashatanyt committed:
make hr centered

natashatanyt committed:
fix broken hyperlink

natashatanyt committed:
Fix more broken hyperlinks

natashatanyt committed:
add hr, fix hyperlinks

natashatanyt committed:
Add more hr

natashatanyt committed:
Change headers and hr

natashatanyt committed:
update `back to table of contents` hyperlinks

natashatanyt committed:
Update hr

natashatanyt committed:
Edit UG

natashatanyt committed:
Merge pull request #328 from natashatanyt/master

Update UG for 2101

sansders committed:
Test new hyperlink

sansders committed:
Add one more hyperlink back to ToC

sansders committed:
Merge pull request #329 from sansders/master

Add hyperlink to UG

sansders committed:
Remove forced page breaks

sansders committed:
Merge pull request #330 from sansders/master

Remove forced page breaks
